From 96eae5f3e81384ab46aeead1f84761bab6e6397d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Wed, 10 Dec 2014 15:58:42 +0000
Subject: [PATCH] Resolves: fdo#87141 Remove deletes the node, release doesn't

regression from introduction of boost::ptr_vector originally one had to Remove
and delete the return to delete it, while now the Remove automatically delete
it. This bare Remove to just "release" it seems to have been overlooked

Change-Id: I175f7c95862ce47402993e5e3b8feaa93af43e75
(cherry picked from commit efcbb12a27a380a442292f17c7f508555a66d4af)
---
 editeng/source/editeng/editundo.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/editeng/source/editeng/editundo.cxx b/editeng/source/editeng/editundo.cxx
index 02989df..3bc5ed3 100644
--- a/editeng/source/editeng/editundo.cxx
+++ b/editeng/source/editeng/editundo.cxx
@@ -189,7 +189,7 @@ void EditUndoDelContent::Redo()
     pEE->RemoveParaPortion(nNode);
 
     // Do not delete node, depends on the undo!
-    pEE->GetEditDoc().Remove( nNode );
+    pEE->GetEditDoc().Release( nNode );
     if (pEE->IsCallParaInsertedOrDeleted())
         pEE->ParagraphDeleted( nNode );
 
-- 
1.9.3

